src/main/java/seedu/duke/Parser.java

```diff
@@ -1,5 +1,9 @@
 package seedu.duke;
 
+import seedu.duke.command.GetCommand;
+import seedu.duke.command.ListCommand;
```

Comment:
This is because we divided work when changing parser. Zhihuang only changed methods that involves GetCommand and ListCommand. Once his merged with mine should be ok alr